index.html

```diff
@@ -224,7 +224,7 @@ <h2>About me</h2>
               <div class="skill">
                 <div class="skill-title">
                   <div class="icon-figure icon-diamond"></div>
-                  <h4>Language</h4>
+                  <h3>Language</h3>
```

Comment:
Kindly wrap `Languages`, `Framework` and `Skills` with `ul` tag and change everyone to `li` tag for accessability

Reply:
My skills are wrapped in `ul` and `li`  [code](https://github.com/hsztan/micro-portfolio/blob/update-accessibility-fix/index.html#L229-L234). Also been researching and found that giving the `ul` a header element is semantically agreed --> [stack overflow...](https://stackoverflow.com/questions/1141639/how-to-semantically-provide-a-caption-title-or-label-for-a-list-in-html)